middlewares/myPassport.js

```diff
@@ -0,0 +1,60 @@
+import passport from 'passport';
+import { Strategy as LocalStrategy } from 'passport-local';
+import { Strategy as JwtStrategy, ExtractJwt } from 'passport-jwt';
+import { Strategy as FacebookStrategy } from 'passport-facebook';
+
+import models from 'models';
+
+import config from 'config';
+
+passport.use(
+	new LocalStrategy(
+		{
+			usernameField: 'username',
+			passwordField: 'password',
+		},
+		async (username, password, cb) => {
+			try {
+				const user = await models.user.findOne({ where: { username } });
+				await models.user.comparePassword(password, user.password);
+				return cb(null, user, { message: 'Logged In Successfully' });
+			} catch (err) {
+				return cb(false, null, { message: 'Wrong login or password' });
+			}
+		}
+	)
+);
+
+passport.use(
```

Comment:
This task is stood before the `passport` involvement, so please rewrite the Token-based auth avoiding `passport`